orange/src/test/java/com/hartwig/hmftools/orange/algo/linx/LinxReportableClustersTest.java

```diff
@@ -24,13 +24,49 @@ public class LinxReportableClustersTest
     @Test
     public void shouldFindAllReportableClusters()
     {
-        LinxData linxData = createTestLinxData();
+        LinxData linxData = linxDataBuilder().build();
         Set<Integer> clusters = LinxReportableClusters.findReportableClusters(linxData);
         assertEquals(new HashSet<>(Set.of(10, 50, 100)), clusters);
     }
 
+    @Test
+    public void shouldNotCountSimpleClusters()
+    {
+        // cluster 10 has single chain link and no exons -> should not be counted
+        LinxData linxData = linxDataBuilder()
+                .clusterIdToChainCount(Map.of(10, 1, 50, 2, 100, 2))
+                .clusterIdToExonCount(Map.of(50, 1, 100, 1))
+                .build();
+        Set<Integer> clusters = LinxReportableClusters.findReportableClusters(linxData);
+        assertEquals(new HashSet<>(Set.of(50, 100)), clusters);
```

Comment:
Is it necessary to create a HashSet from `Set.of`, doesn't that create something that is equal to `clusters` already?

Reply:
It does indeed!